projects/packages/connection/src/class-manager.php

```diff
@@ -1634,6 +1634,7 @@ public function get_authorization_url( $user = null, $redirect = null ) {
 				'site_icon'     => get_site_icon_url(),
 				'site_lang'     => get_locale(),
 				'site_created'  => $this->get_assumed_site_creation_date(),
+				'userless'      => ! $this->has_connected_owner(),
```

Comment:
Out of curiosity, why did you choose to use `! $this->has_connected_owner()` here instead of `$this->is_userless()`?

Reply:
Hi @fgiannar!
In this particular case we're only interested in the fact that there's no master user. We don't care that much if the site is already registered or not, as the `authorization_url` may be retrieved before the site is registered.